docs/analytics/apis/lookup-tables-api.md

```diff
@@ -25,7 +25,14 @@ You can also create and manage lookup tables from the Amplitude web app. See [Lo
 
 ## Considerations
 
-The max file size is 100 MB and the file can't have more than 1,000,000 rows.
+The CSV file must comply with the following requirements:
+
+- The max file size is 100 MB and the file can't have more than 1,000,000 rows.
+- The first row should contain column names/headers.
```

Comment:
Is this a requirement or a recommendation? Will the import fail if no header row is present?

Reply:
it's a requirement, updating..